.github/workflows/build.yml

```diff
@@ -31,4 +31,4 @@ jobs:
       
       - name: Run build script
         run: |
-          python Tools/BuildAddOn.py --configFile config.json --acVersion ${{ matrix.ac-version }}
+          python Tools/BuildAddOn.py -c config.json --acVersion ${{ matrix.ac-version }}
```

Comment:
lets write out the -c as -configFile again, easier to understand for new developers